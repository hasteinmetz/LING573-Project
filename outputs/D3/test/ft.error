Traceback (most recent call last):
  File "/projects/assigned/2122_ling573_elibales/repo/src/fine-tune.py", line 281, in <module>
    main(args)
  File "/projects/assigned/2122_ling573_elibales/repo/src/fine-tune.py", line 199, in main
    train_sentences = preprocess_quotes(train_sentences_raw)
NameError: name 'preprocess_quotes' is not defined
