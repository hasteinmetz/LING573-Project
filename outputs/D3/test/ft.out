(0min 3sec) Using cuda device
Using the GPU:Tesla M10
(0min 3sec) Reading data in from files...

